scrape/scrape.go

```diff
@@ -449,20 +449,11 @@ func mutateSampleLabels(lset labels.Labels, target *Target, honor bool, rc []*re
 		}
```

Comment:
can we change `lset` to `extLset` to be clear? Was quite confused until I figured out those are external labels.

Reply:
It's not the external labels though, they're not involved in scraping. These are the sample labels.